Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -932,9 +932,27 @@ static void OnRemovePlayerMessage(NetworkConnection conn, RemovePlayerMessage ms
                 Destroy(conn.identity.gameObject);
                 conn.identity = null;
             }
+        }
+
+        /// <summary>
+        /// Removes the player object from the connection
+        /// </summary>
+        /// <param name="conn">The connection of the client to remove from</param>
+        /// <param name="destroyServerObject">Indicates whether the server object should be destroyed</param>
+        public static void RemovePlayerForConnection(NetworkConnection conn, bool destroyServerObject)
```

Comment:
is this related to yesterday night's discord discussion about allowing clients to remove players?